### DISCUSSION TITLE
Baseline_Table has no values for a few images

### DISCUSSION THREAD

**Discussion by krkrueger** [gmtsar/gmtsar#840] [Q&A]
The code creating the baseline table is missing some values for a few of the images, this causes my next step of creating the interferograms to fail. Any Idea what could be missing that would cause these to not be present? I deleted the lines from the baseline table so I could move on, but I am windering if there is something I could do to get them to show up?

Thanks!

S1_*_ALL_F3 2014319.5380953117 319 -31.594185929789 -2.612772690283
S1_*_ALL_F3 2014343.5385291467 343
S1_*_ALL_F3 2015002.5385252240 367
S1_*_ALL_F3 2015026.5380688820 391 -66.129416404428 13.256690555863
S1_*_ALL_F3 2015050.5380648114 415 42.407418627610 68.166490341894
S1_*_ALL_F3 2015062.5380644880 427 6.571803716325 17.106380261805
S1_*_ALL_F3 2015086.5380712578 451 -69.619925715950 -54.183776435178
S1_*_ALL_F3 2015110.5382386805 475 -96.421219054315 -96.583401424212

**Comment by Xiaohua-Eric-Xu** [comment]
Could you attache a log file?

You could also use get_baseline_table.csh to produce a new one.

**Reply by krkrueger** [reply]
So I created a new one to look at what the output was, and it looks like this is the problem:

interpolation point outside of data constraints, 
interpolation point outside of data constraints, 

This normally pops up if my DEM is too small, but I am checking the radar images against the DEM and the pins that I selected, and the DEM more than covers the area set up with the pins. Not sure what else could be the issue.  All the other 200 or so images seem to work okay, just not those too.  I plotted them in ArcPro over the DEM, and the DEM doesn't cover the entire radar image, but it does cover the bursts that are inside the pins by a wide margin.

**Reply by Xiaohua-Eric-Xu** [reply]
This feels like an orbit issue. Do you have a data.in file that you use to produce the baseline_table.dat?

**Reply by krkrueger** [reply]
Well, a data.in file is created, and it looks like it has an orbit associated with both of those files with the dates around it making sense. But I am not really sure how to evaluate it after that. How do troubleshoot an orbit issue? Thanks!

**Reply by Xiaohua-Eric-Xu** [reply]
If in your data.in file, there's no .EOF after the data, then it will not be able to process the data from that date. So I asked whether you could share your data.in file for a quick check. Anyways, if you just want to get a correct baseline_table.dat, you could also run `get_baseline_table.csh` to create a new one.
